File: Python ejs/UD03/A02/UD03A023.py
n1=int(input("Introduzca un número: "))
n2=int(input("Introduzca otro número: "))
if n1%n2==0:
    print(f"División exacta. Cociente = {n1/n2}")
else:
    print(f"División no exacta. Cociente = {int(n1/n2)} Resto = {n1%n2}")

File: Java ejs/UD3/A02/UD03A023.java
import java.util.Scanner;
public class UD03A023{
	public static void main(String[] args){
		int n1,n2;
        Scanner teclado=new Scanner(System.in);
        System.out.println("Introduzca un número: ");
        n1=teclado.nextInt();
        System.out.println("Introduzca otro número: ");
        n2=teclado.nextInt();
        if (n1%n2==0)
        {
            System.out.println("Cociente: "+ (n1%n2));
        }else{
            System.out.println("Cociente: "+ (n1%n2));
            System.out.println("Resto: "+ (n1/n2));
        }
	}
}